templates/peering/router/add_edit.html

```diff
@@ -13,6 +13,10 @@
                 {% render_field form.platform %}
                 {% render_field form.encrypt_passwords %}
                 {% render_field form.configuration_template %}
+                {% render_field form.napalm_username %}
```

Comment:
Put this inside a dedicated Bootstrap `card` separate these form fields from the others.